system-tests/test-binary/node_versions_spec.ts

```diff
@@ -9,6 +9,15 @@ function smokeTestDockerImage (dockerImage: string) {
     specDir: 'tests',
     project: 'todos',
   })
+
+  systemTests.it(`can run in ${dockerImage}`, {
```

Comment:
clean up the hierarchy of tests here? right now it's: 

```
e2e binary node versions > can run in ${dockerImage}
e2e binary node versions > can run in ${dockerImage}
```

but something like this would be better:

```
binary node versions > ct > can run in ${dockerImage}
binary node versions > e2e > can run in ${dockerImage}

Reply:
Good catch, addressed in [0654882](https://github.com/cypress-io/cypress/pull/21430/commits/06548825f43c6878abe43d6ff06997d6c8f2b5a0)